### PR TITLE
Added en-doge and ran i18

### PR DESCRIPTION
**Pull request by HonbraDev**
Much bork

### CODE REVIEW

**Comment by HonbraDev** [comment]
image

**Comment by nadirabbas** [comment]
do you know why there are huge changes in other translation files too?

**Comment by HonbraDev** [comment]
@nadirabbas yes, I ran `npm run i18`. If it's a problem, I'll close this one and make a new PR without running i18.

File: kofta/public/locales/en-doge/translation.json [hunk 1]
```diff
@@ -0,0 +1,224 @@
+{
+  "_comment": "if you change this file, do: npm run i18",
```
Comment (HonbraDev) [inline kofta/public/locales/en-doge/translation.json:2]: here

**Comment by nadirabbas** [comment]
what does `i18n` do? doesn't it just add translation key types? You only run `i18n` when you manually add a new translation key. (as far as I know)

**Pull request closed without merging**